assets/vcfanno.toml

```diff
@@ -1,6 +1,6 @@
 # the resources in this file and directory are taken from https://github.com/brentp/vcfanno/blob/master/example
 [[annotation]]
-file="exac.vcf.gz"
+file="variation/exac.vcf.gz"
```

Comment:
Update the module first, I fixed that

Reply:
I updated the module in the last commit, it wasn't fixed...

Reply:
But it works now using this TOML

Reply:
because the resource bundle wasn't fixed, the module and test was

Reply:
Yeah the fixes are already in the pipeline :)